scripts/belyi/find_friends.py

```diff
@@ -16,7 +16,10 @@ def base_field_label(rec):
 
 def genus2_lookup_equation(rec):
     f,h = curve_string_parser(rec)
-    return genus2_lookup_equation_polys(str([f,h]))
+    lab, _ = genus2_lookup_equation_polys(str([f,h]))
+    if lab:
+        return lab
+    return None
```

Comment:
Technically `return None` is unnecessary here since that's the default behavior when there's no return statement.  But it's fine.